src/adafruit_circuitplayground/test/test_express.py

```diff
@@ -19,6 +19,15 @@ def setup_method(self):
         self.pixels = Pixel(self.__state)
         self.__speaker_enabled = False
 
+    def test_acceleration(self):
+        self.cpx._Express__state["motion_x"] = 10
```

Comment:
_&ast;nit&ast;_
Perhaps store the motion values in variables?
Something like:
```
mock_motion_x = 10
.
.
self.cpx._Express__state["motion_x"] = mock_motion_x
.
.
asssert accel[0] = mock_motion_x
```
This way if a developer is working with these values in the future, they will be able to do any changes in a single place.

Reply:
smart, done

Reply:
Done